Dockerfile

```diff
@@ -31,15 +32,14 @@ ENV \
 RUN groupadd --system symbolicator --gid $SYMBOLICATOR_GID \
     && useradd --system --gid symbolicator --uid $SYMBOLICATOR_UID symbolicator
 
-RUN mkdir /etc/symbolicator /data \
-    && chown symbolicator:symbolicator /etc/symbolicator /data
-VOLUME ["/etc/symbolicator", "/data"]
+VOLUME ["/data"]
+RUN mkdir /etc/symbolicator && \
+    chown symbolicator:symbolicator /etc/symbolicator /data
+ONBUILD COPY --chown=symbolicator:symbolicator config.yml /etc/symbolicator/config.yml
 
 EXPOSE 3021
 
-COPY --from=symbolicator-build /usr/local/bin/symbolicator /bin
-# Smoke test
-RUN symbolicator --version && symbolicator --help
+COPY --from=symbolicator-build --chown=symbolicator:symbolicator /usr/local/bin/symbolicator /bin
```

Comment:
Any specific reason why we need that `chown` here?

Reply:
This one is not really necessary. I added it for good hygiene but thinking again, I guess it is actually better to leave it as root?